src/pages/RepoPage/CoverageTab/subroute/FileExplorer/FileListTable/FileListTable.spec.tsx

```diff
@@ -114,12 +113,14 @@ const mockOverview = {
 }
 
 const wrapper =
-  (initialEntries = '/gh/codecov/cool-repo/tree/main/a/b/c') =>
+  (
+    initialEntries = '/gh/codecov/cool-repo/tree/main/a/b/c'
+  ): React.FC<React.PropsWithChildren> =>
   ({ children }) => {
     return (
       <QueryClientProvider client={queryClient}>
         <MemoryRouter initialEntries={[initialEntries]}>
-          <Route path="/:provider/:owner/:repo/tree/:branch/:path+">
+          <Route path="/:provider/:owner/:repo/tree/:branch/:path*">
```

Comment:
Same thing here as previous comment about routes 👀

Reply:
Same response, the tests also don't work with the change.

Reply:
I think it's fine with `*`. Testing on prod, `https://app.codecov.io/gh/codecov/gazebo/tree/main` does resolve properly to the root directory of the repo, so 0 path segments or more should be ok.